src/community/utrecht/alert.tokens.json

```
@@ -0,0 +1,78 @@
+{
+  "utrecht": {
+    "alert": {
+      "icon": {
+        "gap": {
+          "value": "16px"
+        },
+        "error": {
+          "color": {
+            "value": "var(--of-color-danger)"
```

```suggestion
            "value": "{of.color.danger}"
```
CSS syntax is not supported in other output formats, like javascript. Instead, you need to reference it internally and the build tool will output the CSS var.